performance/benchmarks/micro_benchmarks.rb

```diff
@@ -31,6 +32,43 @@ def divider
 
 divider
 
+# Time Threshold: 1 second
+# Setup: Create patient and add many assessments
+patient = Patient.create!(creator: User.first, jurisdiction: Jurisdiction.first, responder: Patient.first)
+micro_results << benchmark(
+  name: 'PatientAssessmentQuerying',
+  time_threshold: 1,
```

Comment:
May want to decrease this further, but let's see how it performs near the end of the PR before changing it.

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/806/commits/c593b41a33decfcecc95ce7cb6d875fadcc59129#diff-eb2f94faa1282056a764ceeae2b4bb03b361611302fd04d7b0ba3b3949573ec4R40